Makefile

```diff
@@ -146,11 +146,11 @@ install: $(RUNTIMES:%=install-%);
 
 install-%:
 	mkdir -p $(PREFIX)/bin
-	$(INSTALL) $(TARGET_DIR)$(TARGET)/$(OPT_PROFILE)/containerd-shim-$*-v1 $(PREFIX)/bin/
+	$(INSTALL) $(TARGET_DIR)$(if $(TARGET),$(TARGET)/,)$(OPT_PROFILE)/containerd-shim-$*-v1 $(PREFIX)/bin/
```

Comment:
Is this still needed after https://github.com/containerd/runwasi/pull/765?

Reply:
I will do a rebase.